allensdk/test/brain_observatory/behavior/behavior_project_cache/test_behavior_project_cloud_api.py

```diff
@@ -4,9 +4,14 @@
 from unittest.mock import MagicMock, create_autospec
 
 from allensdk.api.cloud_cache.manifest import Manifest
-from allensdk.brain_observatory.behavior.behavior_project_cache.project_apis.data_io import behavior_project_cloud_api as cloudapi  # noqa: E501
 from allensdk.brain_observatory.behavior.behavior_project_cache.project_apis.\
-        data_io.behavior_project_cloud_api import MANIFEST_COMPATIBILITY
+    data_io import behavior_project_cloud_api as cloudapi  # noqa: E501
```

Comment:
I don't think you still need the `# noqa: E501`. That was just suppressing a "line too long" error

Reply:
done